traitlets/tests/test_traitlets.py

```diff
@@ -600,23 +589,6 @@ class C(HasTraits):
         c = C()
         self.assertTrue(c.inst is None)
 
-    def test_bad_default(self):
-        class Foo(object): pass
-
-        class A(HasTraits):
-            inst = Instance(Foo)
-
-        self.assertRaises(TraitError, A)
-
-    def test_instance(self):
-        class Foo(object): pass
-
-        def inner():
-            class A(HasTraits):
-                inst = Instance(Foo())
-
```

Comment:
If it requires adding a request for the trait, that's okay, too, but the tests shouldn't be removed altogether.